docs/release/3.4/release-3-4-23.md

```diff
@@ -0,0 +1,224 @@
+OSG Software Release 3.4.23
+===========================
+
+**Release Date**: 2019-01-23
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   Gratia probes 1.20.8
+    -   Interpret CPU expressions for Hosted CEs (Needed for proper accounting)
+    -   Set Processors field properly for Slurm
```

Comment:
We should emphasize that this was a minor change and that records from Slurm sites were fine before and should not be affected after this update